tests/test_protocols/test_fipa.py

```diff
@@ -66,3 +68,57 @@ def test_fipa_propose_serialization():
     p2 = expected_msg.get("proposal")
     assert p1[0].values == p2[0].values
     assert p1[1].values == p2[1].values
+
+
+def test_fipa_accept_serialization():
+    """Test that the serialization for the 'fipa' protocol works."""
+    msg = FIPAMessage(message_id=0, dialogue_id=0, target=0, performative=FIPAMessage.Performative.ACCEPT)
+    msg_bytes = FIPASerializer().encode(msg)
+    envelope = Envelope(to="receiver", sender="sender", protocol_id=FIPAMessage.protocol_id, message=msg_bytes)
+    envelope_bytes = envelope.encode()
```

Comment:
why are we testing envelope encoding here too?

Reply:
Because I keep my tests in different functions, I don't want to mess with @marcofavorito 's code.

Reply:
If you are saying that envelope encoding is only tested here then please move it to the correct test file. Otherwise remove the envelope encoding test. It’s ok to add to @marcofavorito code.